module_utils/helpers.py

```diff
@@ -661,6 +661,22 @@ def get_topology_checksum(console_sock, stateboard):
     ''')
 
 
+def enrich_replicasets_with_leaders(control_console, replicasets):
+    leaders, err = control_console.eval_res_err('''
+        return require('cartridge.failover').get_active_leaders()
+    ''')
+    if err:
+        return err
+
+    if not leaders:
+        return None
+
+    for replicaset_name, replicaset_params in replicasets.items():
+        replicaset_params['leader'] = leaders.get(replicaset_params['uuid'])
```

Comment:
`leader` -> `leader_uuid`. Cartridge replicaset has `leader` member, and it's a dict.